resources/hvac.rb

```diff
@@ -2110,11 +2110,11 @@ def self.apply_dehumidifier(model, runner, energy_factor, integrated_energy_fact
     water_removal_curve = create_curve_biquadratic(model, w_coeff, "DXDH-WaterRemove-Cap-fT", -100, 100, -100, 100)
     energy_factor_curve = create_curve_biquadratic(model, ef_coeff, "DXDH-EnergyFactor-fT", -100, 100, -100, 100)
     part_load_frac_curve = create_curve_quadratic(model, pl_coeff, "DXDH-PLF-fPLR", 0, 1, 0.7, 1)
+    water_removal_rate = UnitConversions.convert(water_removal_rate, "pint", "L")
 
     if energy_factor.nil?
       # shift inputs tested under IEF test conditions to those under EF test conditions with performance curves
-      water_removal_rate_l = UnitConversions.convert(water_removal_rate, "pint", "L")
-      energy_factor, water_removal_rate = dehumidifier_ief_to_ef_inputs(w_coeff, ef_coeff, integrated_energy_factor, water_removal_rate_l)
+      energy_factor, water_removal_rate = dehumidifier_ief_to_ef_inputs(w_coeff, ef_coeff, integrated_energy_factor, water_removal_rate)
```

Comment:
@yzhou601 It's not clear to me why the water removal rate is only adjusted when the energy_factor is nil. Shouldn't it be adjust all the time?

Reply:
Oh it means we didn't get EF inputs directly. Since we require either EF or IEF, it's equivalent to say that we got IEF. In that case we'll need to adjust inputs to be values under EF test conditions.

Reply:
Oh, I see. We're continuing to support EF_80. So we're assuming that if IEF is passed in, water_removal_rate at 65 is also passed in.

Reply:
Yes you got my question, if a person is passing IEF, do you agree to assume all those other inputs are also under 65?

Reply:
I believe the answer is yes. See the glossary at https://www.energystar.gov/products/appliances/dehumidifiers/key_efficiency_criteria, where both IEF and Product Capacity reference the new test spec.

Reply:
Sorry if I missed that question. That might be a question for @shorowit . But in my opinion, it's a little confusing and will depend on where the user is getting information. Maybe there is a better way to distinguish the rated operating conditioned associated with the inputs.

Reply:
No worries at all. True, currently these inputs are got from `extension` elements, if the element under `extension` is`IEF`, it might infer that the `capacity` required in its neighbor is also associated with IEF. But this could be a good suggestion to this hpxml proposal? https://github.com/hpxmlwg/hpxml/pull/208